src/main/java/core/basesyntax/model/Figure.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.model;
+
+public class Figure implements Drawable, FigureArea {
+    private String name;
+    private String color;
+
+    public Figure(String name, String color) {
+        this.name = name;
+        this.color = color;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: "
+                + "name = " + name
+                + ", color = " + color);
+    }
+
+    @Override
+    public double area() {
+        return 0;
```

Comment:
We should not override it here, it could return some illegal value, for example, you create a new Figure and you forgot to override area(), and when area() is called on some instance 0 returned which in not valid value for figure you created